<launch>
  <node  name="perception_node" pkg="perception" type="perception_node" output="screen" >
  <param name="front_rslidar_topic" value="/lslidar_pub1_point_cloud"/>
  <param name="left_rslidar_topic" value="/ns2/rslidar_points"/>
  <param name="right_rslidar_topic" value="/ns1/rslidar_points"/>
  <param name="seg_model" value="3"/>
  <param name="check_angle" value="0"/>
  <param name="pub_ground" value="ground_points"/>
  <param name="pub_no_ground" value="no_ground_points"/>
  </node>

  <node  name="cluster_node" pkg="perception_cluster" type="cluster_node" output="screen">
  <param name="sub_ground" value="ground_points"/>
  <param name="sub_no_ground" value="no_ground_points"/>
  <param name="pub_obst_set" value="obst_set"/>
  <param name="cluster_model" value="3"/>
  <param name="eucliden" value="{minsize:50,maxsize:1000,rounds:3,radius:20,distance:0.2,radius:30,distance:0.3,radius:40,distance:0.4}"/>
  <param name="dbscan" value="{minnum:20,aroundnum:16,radius:0.8}"/>
  </node>

  <node name="track_node" pkg="perception_track" type="track_node" output="screen">
  </node>
</launch>